src/core/load_balancing/rls/rls.cc

```diff
@@ -946,7 +958,7 @@ void RlsLb::ChildPolicyWrapper::ChildPolicyHelper::UpdateState(
     wrapper_->connectivity_state_ = state;
     DCHECK(picker != nullptr);
     if (picker != nullptr) {
-      wrapper_->picker_ = std::move(picker);
+      wrapper_->picker_.swap(picker);
```

Comment:
Please add a comment here that we're not going to unref the picker until we release the lock.

Reply:
Done